registry/server/api_topics.go

```diff
@@ -421,15 +433,15 @@ func (s *Server) DeleteTopicTags(ctx context.Context, req *pb.TopicRequest) (*pb
 }
 
 // fetchTopicSet fetches metadata for all topics.
-func (s *Server) fetchTopicSet(req *pb.TopicRequest) (TopicSet, error) {
-	topicRegex := []*regexp.Regexp{}
+func (s *Server) fetchTopicSet(params fetchTopicSetParams) (TopicSet, error) {
+	var topicRegex = []*regexp.Regexp{}
```

Comment:
Both work, it could be considered more conventional to use the long form assignment when you're not taking assigning literals or taking the return value from a func.

Reply:
I was wondering more why you changed it from the initial `topicRegex := []*regexp.Regexp{}`, but I guess it's more of a `[nit]`.